core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/compress/DirectCompressCodec.java

```diff
@@ -347,9 +347,7 @@ private void fillVector(ColumnPage columnPage, CarbonColumnVector vector,
             columnPage.getNullBits());
       } else if (vectorDataType == DataTypes.FLOAT) {
         float[] floatPage = columnPage.getFloatPage();
-        for (int i = 0; i < pageSize; i++) {
-          vector.putFloats(0, pageSize, floatPage, 0);
-        }
+        vector.putFloats(0, pageSize, floatPage, 0);
```

Comment:
Line numbers from 322 to 334 also remove as it is duplicated.

Reply:
removed